db/migrate/20160930150810_add_smart_proxy_download_policy.rb

```diff
@@ -0,0 +1,14 @@
+class AddSmartProxyDownloadPolicy < ActiveRecord::Migration
+  def up
+    #set default to on_demand, but update existing proxies to inherit
+    add_column :smart_proxies, :download_policy, :string, :default => ::Runcible::Models::YumImporter::DOWNLOAD_ON_DEMAND
```

Comment:
This is a good point for discussion, and I could see arguments either way.  In the current code that setting is only the default for new Repositories (doesn't really affect the capsule as a whole), so I didn't initially think about it as a default for the capsules.  I could also see someone wanting the default for Repositories to be different than the default for capsules (although maybe a default for capsules is less important).  Would it make sense to add a 2nd setting for the capsule default?